code/go/0chain.net/smartcontract/benchmark/main/cmd/options.go

```diff
@@ -12,6 +24,28 @@ import (
 	"0chain.net/smartcontract/benchmark/main/cmd/log"
 )
 
+var benchmarkSources = map[bk.Source]func(data bk.BenchData, sigScheme bk.SignatureScheme) bk.TestSuite{
+	bk.Storage:         storagesc.BenchmarkTests,
+	bk.StorageRest:     storagesc.BenchmarkRestTests,
+	bk.Miner:           minersc.BenchmarkTests,
+	bk.MinerRest:       minersc.BenchmarkRestTests,
+	bk.Faucet:          faucetsc.BenchmarkTests,
+	bk.FaucetRest:      faucetsc.BenchmarkRestTests,
+	bk.Vesting:         vestingsc.BenchmarkTests,
+	bk.VestingRest:     vestingsc.BenchmarkRestTests,
+	bk.MultiSig:        multisigsc.BenchmarkTests,
+	bk.ZCNSCBridge:     zcnsc.BenchmarkTests,
+	bk.ZCNSCBridgeRest: zcnsc.BenchmarkRestTests,
+	bk.Control:         control.BenchmarkTests,
+}
+
+func readEventDbTests(filename string) (map[string][]event.Event, error) {
+	file, _ := ioutil.ReadFile(filename)
```

Comment:
why do we have file here, we need to execute events from transaction execution

Reply:
It adds flexibility at no extra cost.